.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,7 +4,15 @@
 <!-- Explanation of the change or anything fishy that is going on -->
 
 ### Fixed Issues
-<!-- Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing -->
+<!---
+Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing.
+Do NOT add the special GH keywords like `fixed` etc, we have our own process of managing the flow.
+It MUST be an entire link to the issue, otherwise the linking will not work as expected.
```

Comment:
I'm not 100% sure, but I think usually there is a comma after 'otherwise'. Grammarly suggest putting it like:

```
It MUST be an entire link to the issue; otherwise, the linking will not work as expected.
```

I also found this sentence online that looks similar in structure: `The sick woman needs to be given treatment; otherwise, her illness will become fatal.`

Reply:
@mallenexpensify Could we please get your 👀 on this copy. More eyes will know better.

Reply:
That looks good to me 👍

Reply:
Perfect! Updated!